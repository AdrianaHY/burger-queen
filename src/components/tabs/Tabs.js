import React, { Component } from 'react';

import breakfastIcon from '../../assets/breakfast.svg';
import lunchAndDinner from '../../assets/burger-and-soda.svg';

import './Tabs.scss';

class Tabs extends Component {
	render() {
		const { handlerMenuOption } = this.props;
		return (
			<div className="tabs-container">
				<button onClick={handlerMenuOption} className="tab">
					<img src={breakfastIcon} alt="breakfast" className="menu-image"/>
					<p className="menu-title">breakfast</p>
				</button>
				<button onClick={handlerMenuOption} className="tab">
					<img src={lunchAndDinner} alt="breakfast" className="menu-image"/>
					<p className="menu-title">lunch and dinner</p>
				</button>
			</div>
		)
	}
}

export default Tabs;