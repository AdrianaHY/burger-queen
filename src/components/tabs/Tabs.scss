@import '../../styles/_variables.scss';

.tabs-container {
	display: flex;
	align-items: center;
	justify-content: space-evenly;

	.tab {
		border-bottom: 3px solid $yellow;
		border-top: none;
		border-left: none;
		background: transparent;
		width: 100%;

		&.active {
			border-bottom: 3px solid $lightYellow;
		}
	}

	.menu-image {
		height: 100px;
	}

	.menu-title {
		color: $purple;
		font-family: $fontFamily;
		font-size: $fontSizeHuge;
		margin-top: 0;
	}
}