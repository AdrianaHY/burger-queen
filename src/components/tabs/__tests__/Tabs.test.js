import React from 'react';
import { shallow } from 'enzyme';
import Tabs from '../Tabs';

describe('components | tabs | Tabs', () => {
	it('Renders the component', () => {
		const props = {
			handlerMenuOption: jest.fn()
		};

		const component = shallow(
			<Tabs {...props} />
		);

		expect(component).toMatchSnapshot();
	})
})