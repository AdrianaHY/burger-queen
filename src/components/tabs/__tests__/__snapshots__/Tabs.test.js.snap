// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | tabs | Tabs Renders the component 1`] = `ShallowWrapper {}`;
